_includes/pbc/all/install-features/202304.0/install-the-gift-cards-feature.md

```diff
@@ -0,0 +1,1338 @@
+     *
+     * @return array
+     */
+    public function getGiftCardOnlyShipmentMethods(): array
+    {
+        return [
+            ShipmentConfig::SHIPMENT_METHOD_NAME_NO_SHIPMENT,
+        ];
+    }
+}
+```
+
+{% info_block warningBox "Verification" %}
+
+When you finish the *Setup Behaviour* step, make sure that the `NoShipment` shipment method is selected automatically while ordering only a gift card.
+
+{% endinfo_block %}
+
+#### Gift Card usage process
+
+Extend your project with the following configuration.
+
+**config/Shared/config_default.php**
+
+```php
+<?php
+
+use Spryker\Shared\Kernel\KernelConstants;
+use Spryker\Shared\Nopayment\NopaymentConfig;
+use Spryker\Shared\Nopayment\NopaymentConstants;
+use Spryker\Shared\Oms\OmsConstants;
+use Spryker\Shared\Sales\SalesConstants;
+use Spryker\Zed\GiftCard\GiftCardConfig;
+
+// ---------- Dependency injector
+$config[KernelConstants::DEPENDENCY_INJECTOR_ZED] = [
+    'Oms' => [
+        GiftCardConfig::PROVIDER_NAME,
+    ],
+];
+
+$config[NopaymentConstants::NO_PAYMENT_METHODS] = [
+    NopaymentConfig::PAYMENT_PROVIDER_NAME,
+];
+$config[NopaymentConstants::WHITELIST_PAYMENT_METHODS] = [
+    GiftCardConfig::PROVIDER_NAME,
+];
+
+// ---------- State machine (OMS)
+$config[OmsConstants::ACTIVE_PROCESSES] = [
+    'Nopayment01',
+];
+$config[SalesConstants::PAYMENT_METHOD_STATEMACHINE_MAPPING] = [
+    GiftCardConfig::PROVIDER_NAME => 'DummyPayment01', // Order State Machine name of your choice
+    NopaymentConfig::PAYMENT_PROVIDER_NAME => 'Nopayment01',
+];
+```
+
+**src/Pyz/Zed/GiftCard/GiftCardConfig.php**
+
+```php
+<?php
+
+namespace Pyz\Zed\GiftCard;
+
+use Spryker\Shared\DummyPayment\DummyPaymentConfig;
+use Spryker\Zed\GiftCard\GiftCardConfig as SprykerGiftCardConfig;
+
+class GiftCardConfig extends SprykerGiftCardConfig
+{
+    /**
+     * Provides a list of payment method names that are disabled to use when the quote contains gift card item(s) to purchase.
+     *
+     * @return array
+     */
+    public function getGiftCardPaymentMethodBlacklist(): array
+    {
+        return [
+            DummyPaymentConfig::PAYMENT_METHOD_INVOICE,
+        ];
+    }
+}
+```
+
+**src/Pyz/Zed/Sales/SalesConfig.php**
+
+```php
+<?php
+
+namespace Pyz\Zed\Sales;
+
+use Spryker\Zed\Sales\SalesConfig as SprykerSalesConfig;
+
+class SalesConfig extends SprykerSalesConfig
+{
+    /**
+     * @return array
+     */
+    public function getSalesDetailExternalBlocksUrls()
+    {
+        $projectExternalBlocks = [
+            'giftCards' => '/gift-card/sales/list', // lists used gift cards for the order
+        ];
+
+        $externalBlocks = parent::getSalesDetailExternalBlocksUrls();
+
+        return array_merge($externalBlocks, $projectExternalBlocks);
+    }
+}
+```
+
+{% info_block warningBox "Verification" %}
+When you finish the Setup Behaviour step, make sure the following:
```

Comment:
Can you add the link to the step you are mentioning—the *Setup Behaviour step*

Reply:
@ilyakubanov  can you confirm my changes here
`[Setup Behaviour](#install-the-required-modules-using-composer)`
?